src/coreclr/vm/methodtablebuilder.h

```diff
@@ -2073,7 +2073,7 @@ class MethodTableBuilder
 
     // --------------------------------------------------------------------------------------------
     // Used for analyzing overlapped fields defined by explicit layout types.
-    enum bmtFieldLayoutTag {empty, nonoref, oref, byref};
+    enum bmtFieldLayoutTag : BYTE {empty, nonoref, oref, byref};
```

Comment:
It would be also nice to change the type of `BYTE *pFieldLayout` and similar argument of related methods to `bmtFieldLayoutTag*`.

Reply:
Good idea. Done.